src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
+    return findByXPathOrCSS(String
+            .format("(//*[contains(text(),'%s')]//following::p[contains(@class,'likersNumber')])[1]", activity));
+  }
+
+  public void clickReactionList(String activity) {
+    getReactionActivityList(activity).clickOnElement();
+  }
+
+  @FindBy(xpath = "//*[contains(@class,'v-list-item__content drawerTitle') and contains (text(),'Reactions')]")
+  private BaseElementFacade reactionDrawer;
+
+  public void drawerDisplay() {
+    reactionDrawer.shouldBeVisible();
+  }
+
+  private BaseElementFacade getUserInDrawerListLikers(String user) {
+    return findByXPathOrCSS(
+            String.format("//aside[contains(@class,'v-navigation-drawer--open')]//p[contains(text(),'%s')]", user));
+  }
+
+  public void hoverAndGoToUSerProfile(String user) {
+    getUserInDrawerListLikers(user).hover(
+            String.format("//aside[contains(@class,'v-navigation-drawer--open')]//p[contains(text(),'%s')]", user));
```

Comment:
Use simple hover method without parameters to hiver on selected element.

```suggestion
    getUserInDrawerListLikers(user).hover();
```

Reply:
done